### PR TITLE
Update component to use new versions of text, avatar, stack and foundations

### PR DESCRIPTION
**Pull request by JanuarAsprilla**
here the component is re-factored to use the new version of avatar, stack and foundations

### CODE REVIEW

File: src/User/index.tsx [hunk 1]
```diff
@@ -1,27 +1,25 @@
 import { Avatar } from "@inubekit/avatar";
 import { Text } from "@inubekit/text";
 import { Stack } from "@inubekit/stack";
-import { inube } from "@inubekit/foundations";
 import { Size } from "./props";
 
-const { spacing } = inube;
 export interface IUserProps {
```
Comment (cmarin001) [inline src/User/index.tsx:6]: It would be preferable to rename it simply as IUser.

**Review by cmarin001 — CHANGES_REQUESTED** [review]

File: package.json [hunk 2]
```diff
@@ -35,10 +35,12 @@
     "postversion": "git push --follow-tags && npm run auto release"
   },
   "dependencies": {
-    "@inubekit/avatar": "^1.0.0",
-    "@inubekit/foundations": "^1.1.2",
+    "@inubekit/avatar": "^2.0.0",
+    "@inubekit/foundations": "^2.5.0",
     "@inubekit/stack": "^1.1.0",
```
Comment (wfercanas) [inline package.json:40]: Install latest version of `stack`

File: src/User/index.tsx [hunk 3]
```diff
@@ -1,27 +1,25 @@
...
   return (
-    <Stack justifyContent="flex-start" alignItems="center" gap={spacing.s200}>
+    <Stack justifyContent="flex-start" alignItems="center" gap="16px">
       {size === "large" && (
         <Stack
           direction="column"
           justifyContent="center"
           alignItems="center"
-          gap={spacing.s050}
+          gap="8px"
```
Comment (wfercanas) [inline src/User/index.tsx:22]: This gap should be of `4px`

**Review by wfercanas — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (JanuarAsprilla) [reply package.json:40]: solved

File: src/User/index.tsx [hunk 4]
```diff
@@ -1,27 +1,25 @@
 import { Avatar } from "@inubekit/avatar";
 import { Text } from "@inubekit/text";
 import { Stack } from "@inubekit/stack";
-import { inube } from "@inubekit/foundations";
 import { Size } from "./props";
 
-const { spacing } = inube;
-export interface IUserProps {
-  userName: string;
+export interface IUser {
```
Comment (cmarin001) [inline src/User/index.tsx:6]: The export statement for this interface needs to be positioned at the end of the file, and it should be formatted as follows: export type { IUser }

**Review by cmarin001 — CHANGES_REQUESTED** [review]

File: src/User/User.stories.tsx [hunk 5]
```diff
@@ -11,24 +9,11 @@ const story = {
   argTypes: props,
 };
 
-export const Default = (args: IUserProps) => <User {...args} />;
+export const Default = (args: IUser) => <User {...args} />;
```
Comment (cmarin001) [inline src/User/User.stories.tsx:12]: the export should be at the end of the file

**Review by cmarin001 — CHANGES_REQUESTED** [review]

File: src/User/index.tsx (hunk 4 above)
Comment (JanuarAsprilla) [reply src/User/index.tsx:6]: solved

File: package.json (hunk 2 above)
Comment (wfercanas) [reply package.json:40]: The package.json still does the installation of stack v1.1.0, while the lastest version of the component is v2.0.0

Comment (JanuarAsprilla) [reply package.json:40]: solved

**Review by wfercanas — APPROVED** [review]

**Pull request merged**